kitsune/questions/api.py

```diff
@@ -289,6 +302,29 @@ def take(self, request, pk=None):
 
         return Response(status=204)
 
+    @action(methods=['POST'], permission_classes=[permissions.IsAuthenticated])
+    def add_tag(self, request, pk=None):
+        question = self.get_object()
+        tag = request.DATA['tag']
```

Comment:
If you do this, it will give a 50x error if the user doesn't pass a tag. It is better to do something like in `delete_metadata` to check that it is the data, and if not, give a 400 bad request error. 500s are bad because the are completely opaque to users.